src/amo/pages/Home/styles.scss

```diff
@@ -114,11 +128,24 @@
   height: 64px;
   margin-bottom: 5px;
   transform: scale(0.8);
-  width: 100%;
+  width: 35%;
 
   @include respond-to(medium) {
     transform: initial;
   }
+
+  @include respond-to(large) {
+    background-position: center;
+    width: 100%;
+  }
+
+  [dir='rtl'] & {
+    background-position: right;
+
+    @include respond-to(large) {
+      background-position: center;
```

Comment:
why do you need to set this again? It is set by the previous block line 138.

Reply:
because the `rtl` overwrites the center background position and therefore I have to set it to center in the `rtl` mode also for large viewports.